scripts/testnet.sh

```diff
@@ -45,17 +45,53 @@ get_rpc() {
 # $2=rpc
 wallet_balance() {
   local address=${1}
-  local rpc=${2}
+  local rpc=${2} # @TODO: Replace RPC for chain to avoid calling only xdai
 
-  yarn run --silent ethers eval "new ethers.providers.JsonRpcProvider('${rpc}').getBalance('${address}').then(b => formatEther(b))"
+  curl https://api.hoprnet.org/api/balance/xdai/$1/native/get?text=true
```

Comment:
Instead, shouldn't we use our provider url here, pass it to the API and have the API use it? E.g. via GET parameter `?provider=https://..`

Reply:
Agreed, we should refactor the API to take this value and know what to fund based on that.